webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -77,12 +77,10 @@ <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signatur
             <li class="ui-state-default ui-corner-top">
               <a href="#extensions" class="ui-tabs-anchor"><span>Extensions</span></a>
             </li>
-            {% if raw.TelemetryEnvironment %}
-              <li class="ui-state-default ui-corner-top">
-                <a href="#telemetryenvironment" class="ui-tabs-anchor"><span>Telemetry Environment</span></a>
-              </li>
-            {% endif %}
-            <li class="ui-state-default ui-corner-top correlations hidden">
+            <li class="ui-state-default ui-corner-top">
+              <a href="#telemetryenvironment" class="ui-tabs-anchor"><span>Telemetry Environment</span></a>
+            </li>
+            <li class="ui-state-default ui-corner-top">
```

Comment:
This changes the correlations tab to always show. If there's no correlations data, then the page will say there's no data available.